cmd/kops/create_cluster.go

```diff
@@ -760,19 +765,34 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		cluster.Spec.API = &api.AccessSpec{}
 	}
 	if cluster.Spec.API.IsEmpty() {
-		switch cluster.Spec.Topology.Masters {
-		case api.TopologyPublic:
-			cluster.Spec.API.DNS = &api.DNSAccessSpec{}
```

Comment:
I guess that's a good point, however by setting Topology to private this was already the case. Any thoughts on the best way to handle this?